apps/db/lib/db/customer.ex

```diff
@@ -0,0 +1,31 @@
+defmodule DB.Customer do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  @cast_fields [
+    :user_id, :name, :connected_message_id, :answer,
+    :welcome_message_id, :lang, :attempts,
+  ]
+  @required_fields [
+    :user_id, :name, :connected_message_id, :answer,
+    :lang, :attempts,
+  ]
+  schema "customers" do
```

Comment:
Why you call it Customer?) Not a User?

Reply:
Because I've reserved user table name for chat administration. It seems, that user is wrong name for entity which contains one user many times, that differs by chatname.
Mb customer is not good also, but still...

Reply:
:+1: